config/jobs/kubernetes-sigs/sig-windows/windows-unit.yaml

```diff
@@ -42,6 +42,9 @@ presubmits:
   kubernetes/kubernetes:
     - name: pull-kubernetes-unit-windows-master
       always_run: false
+      branches:
```

Comment:
FWIW we usually express this as ~~skip_branches release-.*, this seems OK though

Reply:
This is how some of the other sig-windows ones are.
I can update if needed